solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -574,306 +584,124 @@ describe("audius-data", function () {
-      baseAuthorityAccount,
-      handleBytesArray,
-      bumpSeed,
-      userAuthorityDelegateKeypair.publicKey,
-      addUserAuthorityDelegateArgs
-    );
-
     const acctState = await program.account.userAuthorityDelegate.fetch(
-      userAuthorityDelegatePDA
+      delegate.userAuthorityDelegatePDA
     );
     const userStoragePdaFromChain = acctState.userStorageAccount;
     const delegateAuthorityFromChain = acctState.delegateAuthority;
     expect(userStoragePdaFromChain.toString(), "user storage pda").to.equal(
-      newUserAcctPDA.toString()
+      delegate.userAccountPDA.toString()
     );
     expect(
-      userAuthorityDelegateKeypair.publicKey.toString(),
+      delegate.userAuthorityDelegateKeypair.publicKey.toString(),
       "del auth pda"
     ).to.equal(delegateAuthorityFromChain.toString());
     const updatedCID = randomCID();
     await updateUser({
       program,
       metadata: updatedCID,
-      userStorageAccount: newUserAcctPDA,
-      userAuthorityKeypair: userAuthorityDelegateKeypair,
-      userAuthorityDelegate: userAuthorityDelegatePDA,
-      authorityDelegationStatusAccount: authorityDelegationStatusPda,
+      userStorageAccount: delegate.userAccountPDA,
+      userAuthorityKeypair: delegate.userAuthorityDelegateKeypair,
+      userAuthorityDelegate: delegate.userAuthorityDelegatePDA,
+      authorityDelegationStatusAccount: delegate.authorityDelegationStatusPDA,
     });
     const removeUserAuthorityDelegateArgs = {
       accounts: {
         admin: adminStorageKeypair.publicKey,
-        user: newUserAcctPDA,
-        userAuthorityDelegatePda: userAuthorityDelegatePDA,
-        userAuthority: newUserKeypair.publicKey,
+        user: delegate.userAccountPDA,
+        newUserAuthorityDelegate: delegate.userAuthorityDelegatePDA,
+        signerUserAuthorityDelegate: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatus: delegate.authorityDelegationStatusPDA,
+        authority: delegate.newUserKeypair.publicKey,
         payer: provider.wallet.publicKey,
         systemProgram: SystemProgram.programId,
       },
-      signers: [newUserKeypair],
+      signers: [delegate.newUserKeypair],
     };
 
-    console.log(`Removing delegate authority ${userAuthorityDelegatePDA}`);
     await program.rpc.removeUserAuthorityDelegate(
-      baseAuthorityAccount,
-      handleBytesArray,
-      bumpSeed,
-      userAuthorityDelegateKeypair.publicKey,
-      userAuthorityDelegateBump,
+      delegate.baseAuthorityAccount,
+      delegate.handleBytesArray,
+      delegate.userBumpSeed,
+      delegate.userAuthorityDelegateKeypair.publicKey,
+      delegate.userAuthorityDelegateBump,
       removeUserAuthorityDelegateArgs
     );
 
     // Confirm account deallocated after removal
-    await pollAccountBalance(provider, userAuthorityDelegatePDA, 0, 100);
+    await pollAccountBalance(
+      provider,
+      delegate.userAuthorityDelegatePDA,
+      0,
```

Comment:
can we assign named variables for these numbers so it's clear what the params are?

Reply:
in this specific case I think it's actually better as 0 - poll until account balance is 0, retry 100 times

Reply:
unless we have a func like pollAccountEmpty or smth?

Reply:
oh yeah i just mean before calling the fn add `const accountBalanceToPollUntil = 0` `const retryCount = 100` and then call `await pollAccountBalance(provider, userAuthorityDelegatePDA, accountBalanceToPollUntil, retryCount)` instead of passing in the numbers which makes the reader have to look up the fn definition if they're not familiar.